packages/website/docs/autocomplete-js.md

```diff
@@ -343,5 +343,27 @@ const {
   setStatus,
   setContext,
   refresh,
+  update,
+  destroy,
 } = autocomplete(options);
 ```
+
+## Helpers
+
+### `refresh`
+
+> `() => void`
+
+Updates the UI state. You must call this function whenever you mutate the state with setters.
+
+### `update`
+
+> `(updatedOptions: Partial<AutocompleteOptions<TItem>>) => void`
+
+Updates the Autocomplete experience with new options.
```

Comment:
```suggestion
Updates the Autocomplete instance with new options.
```

Reply:
Copied that from TSDoc, we might want to update it as well then.